src/main/resources/lib/config.js

```diff
@@ -0,0 +1,37 @@
+/*global app, require*/
+
+const portal = require('/lib/xp/portal');
+const admin = require('/lib/xp/admin');
+
+const getConfig = () => {
+    const csToolUri = admin.getToolUrl('com.enonic.app.contentstudio', 'main');
```

Comment:
We should move 'com.enonic.app.contentstudio' to a const, to avoid repeating it several times.